src/schema/schema_create.c

```diff
@@ -961,6 +977,103 @@ __create_data_source(
     return (dsrc->create(dsrc, &session->iface, uri, (WT_CONFIG_ARG *)cfg));
 }
 
+/*
+ * __import_entry_cmp --
+ *     Qsort function: sort the import entries array by name.
+ */
+static int WT_CDECL
+__import_entry_cmp(const void *a, const void *b)
```

Comment:
Static functions in a file should use a consistent prefix for all in that file. I suggest calling this function `__create_import_cmp`.

Reply:
Agree, done.